### ISSUE TITLE
Improve Documentation

Small improvements to make:

### ISSUE THREAD

**Issue by NicolasKlenert** [NicolasKlenert/enterpolation#28]
- [ ] Add `.gen()` to the documentation to make it more visible

**Comment by Wilker-uwu** [comment]
`gen()`, `extract()`, `sample()` and `slice()` would be good together :3

**Comment by NicolasKlenert** [comment]
Yeah it's probably a good idea to create an own section and go through the most used methods one can call on curves
